docs/topics/exceptions.md

```diff
@@ -90,8 +90,8 @@ as well as [for Swift and Objective-C](native-objc-interop.md#errors-and-excepti
 val s = person.name ?: throw IllegalArgumentException("Name required")
 ```
 
-The type of the `throw` expression is the special type `Nothing`.
-The type has no values and is used to mark code locations that can never be reached.
+The `throw` expression is the special `Nothing` type.
```

Comment:
I don't think we can say that 'expression is.. Nothing type'. It should be 'expression has type Nothing' or 'expression is of Nothing type'